utils/measure/measure/util/measure_util.py

```diff
@@ -44,14 +141,21 @@ def take_measurement(
         retry_count: int = 0,
     ) -> float:
         """Get a measurement from the powermeter, take multiple samples and calculate the average"""
+
+        if self.config.max_retries is None:
+            _LOGGER.error("No max_retries value was configured. Setting the default of 5 to prevent infinite loop.")
+            self.config.max_retries = 5
+
```

Comment:
```suggestion
```

This can be removed as 5 is the default anyway when requesting `self.config.max_retries`, that won't return `None` in any case.
Also `self.config.max_retries = 5` could never work as it has no setter.
My IDE also complains about it.